src/libs/scrollController/windowScrolling.js

```diff
@@ -0,0 +1,38 @@
+const privateHandleScroll = Symbol('handleScroll');
+
+class WindowScrolling {
+    constructor() {
+        this.callback = null;
+        this.listening = false;
+        this.isTicking = false;
+        this[privateHandleScroll] = this[privateHandleScroll].bind(this);
+    }
+
+    [privateHandleScroll](event) {
+        if (!this.isTicking) {
+            window.requestAnimationFrame(() => {
+                this.callback(this.callback(this, event));
+                this.isTicking = false;
+            });
+            this.isTicking = true;
+        }
+    }
+
+    startListening(callback) {
+        this.callback = callback;
+        window.addEventListener('scroll', this[privateHandleScroll]);
+        window.addEventListener('wheel', this[privateHandleScroll]);
+        this.listening = true;
+    }
+
+    stopListening() {
+        if (this.listening) {
+            this.listening = false;
+            window.removeEventListener('scroll', this[privateHandleScroll]);
+            window.removeEventListener('wheel', this[privateHandleScroll]);
+            this.callback = null;
+        }
+    }
+}
```

Comment:
there is an issue when trying to scroll inside the picklist to see other options, it will close the menu when the intention is scrolling down inside the menu

Reply:
ohh... yess. i forgot to avoid that one...